app/Ptr/DnsRecordService.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Packages\Rdns\App\Ptr;
+
+class DnsRecordService {
+  
+  /**
+   * @param string $domain
+   * 
+   * @return array 
+   */
+  public function get($domain){
```

Comment:
we should call this something more specific like `getARecords` since we are filtering to A/AAAA records